spread.yaml

```diff
@@ -816,6 +824,42 @@ suites:
         restore: |
             "$TESTSLIB"/prepare-restore.sh --restore-suite
 
+    tests/nested/classic-preseed/:
```

Comment:
perhaps we could have a more generic test suite which allows the test to create/start/stop/destroy the vm, so then we could use that for more specific use cases. We could have something like tests/nested/manual instead and there the test should take care of the nested instance. @stolowski does is make sense to you?
You should need just to move the create_nested_classic_vm to the preparation of the test and the destroy_nested_vm to the restore. Also it should be available for all the systems and the preseed test should be available for ubuntu-19.10-64 and ubuntu-20.04-64

Reply:
I think it's a great idea, done.